core/env/spiffeid_test.go

```diff
@@ -110,17 +111,17 @@ func TestWorkloadSpiffeIdPrefix(t *testing.T) {
 	}{
 		{
 			name: "default_safe_spiffeid_prefix",
-			want: "spiffe://vsecm.com/workload/",
+			want: string(env.VSecMSpiffeIdPrefixWorkloadDefault),
 		},
 		{
 			name: "safe_spiffeid_prefix_from_env",
 			setup: func() error {
-				return os.Setenv("VSECM_SPIFFEID_PREFIX_WORKLOAD", "spiffe://vsecm.com/workload/test/")
+				return os.Setenv("VSECM_SPIFFEID_PREFIX_WORKLOAD", "spiffe://vsecm.com/workload/test/ns/test/sa/test/n/test")
```

Comment:
Thanks to the stricter SPIFFE ID validation, we had to update the tests to be stricter too. – the test do not pass if the ids do not conform to a certain pattern.